src/firefox/data/scripts/firefox_browser_api.ts

```diff
@@ -52,6 +52,9 @@ class FirefoxBrowserApi implements BrowserAPI {
   }
 
   public showNotification = (notificationText :string) => {
-    // TODO implement!
+    port.emit('showNotification', {
+      text: notificationText,
+      iconURL: './icons/128_online.png'
```

Comment:
might be worth adding a comment or defining it as a UI constant so it's in the same place as chrome.